src/br/com/ciandt/mhiroshi/service/ComentarioService.java

```diff
@@ -0,0 +1,37 @@
+package br.com.ciandt.mhiroshi.service;
+
+/**
+ * classe de servo�o para comentarios
+ * @author mhiroshi
+ * 
+ */
+public class ComentarioService {
+
+	private final static String COMENTARIO_REMOVIDO = new String("Coment�rio removido!"); 
+	
+public ComentarioService() {
+	
+}
+	/**
+	 * metodo para adicionar um 'like'
+	 * @param comentario - {@link If} variavel boolean
```

Comment:
O @link faz referência a uma classe já existente.
```suggestion
	 * @param comentario - {@link String} comentário recebido como texto
```